tests/utils/signature.py

```diff
@@ -0,0 +1,14 @@
+from cryptography.verify_signature import V_OFFSET
+
+
+def compute_vs(v: int, s: str) -> str:
```

Comment:
Let's move `compute_vs` to `recover_vs`
Even if compute_vs is used only for tests

Reply:
Done